### ISSUE TITLE
Adding a 'none' category

### ISSUE THREAD

**Issue by lcrmorin** [hms-dbmi/UpSetR#175]
Hello,

The tool is quite practical but for my use case it lacks an 'empty' category : a way to display the population that is in no set.

One option would be to build a new category like 'none', but it would appears on top of existing categories, and show a black dot for the population that appears in 'none'.

I am considering implementing a solution that would take an optional population or an optional count, to display it on top of grey dots.

Would this make sense ? would this be welcome ?

**Comment by jdhenaos** [comment]
Maybe you are looking for something similar at #109 

Regards,

Juan

**Comment by lcrmorin** [comment]
Yes, this is what I was looking for. As the author says : "That does help as an interim solution, but I would still enjoy seeing the inclusion of the empty set as a feature."

**Comment by jergosh** [comment]
FWIW, seconded. It would be really helpful to have a column (referring to the plot here: https://github.com/hms-dbmi/UpSetR/issues/109#issuecomment-380642611) with all circles greyed out but without the additional row of 'nulls'.

**Comment by RichardJActon** [comment]
FYI I you can do empty sets in the `complexheatmap` package's implementation of upset plots:

https://jokergoo.github.io/ComplexHeatmap-reference/book/upset-plot.html
